.gitignore

```diff
@@ -2,11 +2,10 @@
 composer.phar
 output
 vendor
-
+tags
```

Comment:
oh sorry, in vim I use a plugin called exhuberant ctags, it creates an index on disk for all symbols in the project which allows me to do refactoring, jump to source etc.  makes vim feel like an IDE.  its the equivalent of the .idea directory you have with phpstorm